recipes/nsis/bld.bat

```diff
@@ -0,0 +1,4 @@
+robocopy . "%PREFIX%" /S /XF bld.bat
+if errorlevel 8 exit 1
```

Comment:
I believe this should be [greater than or equal to 8](http://ss64.com/nt/robocopy-exit.html)